### DISCUSSION TITLE
p/debuginn-hugo-blog/

### DISCUSSION THREAD

**Discussion by giscus** [debuginn/blog#122] [General]
# debuginn-hugo-blog/

经历了三个月的整体站点迁移，数据的修复，网站的搭建，目前全站代码已经完成 github 代码托管部署，主站点重新调整设计，以后就不需要关心网站的动态部署问

https://blog.debuginn.cn/p/debuginn-hugo-blog/

**Comment by halohsu** [comment]
我也终于学会了 Hugo 主题开发，┭┮﹏┭┮ 呜呜。

**Reply by debuginn** [reply]
好家伙 logo 中无时无刻不彰显你对股市的渴望 哈哈哈

**Comment by halohsu** [comment]
域名还在备案 哈哈哈
